opm/material/localad/Evaluation.hpp

```diff
@@ -419,9 +421,9 @@ std::ostream& operator<<(std::ostream& os, const Evaluation<Scalar, VarSetTag, n
 } // namespace LocalAd
 } // namespace Opm
 
-// make the Dune matrix/vector classes happy. Obviously, this is not very elegant...
-
-#ifdef DUNE_DENSEMATRIX_HH
+// In Dune 2.3, the Evaluation.hpp header must be included before the fmatrix.hh
```

Comment:
I think this (and the following, unchanged) comment should be changed to a short explanation of what goes wrong, as it stands it is not fully correct.